packages/realm/src/schema/validate.ts

```diff
@@ -0,0 +1,162 @@
+////////////////////////////////////////////////////////////////////////////
+//
```

Comment:
This file does not need to be reviewed. The content is merely refactored from `Configuration.ts` into this `/schema/validate.ts` file.